### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by kiranpatel11** [paketo-buildpacks/new-relic#9]
Can you enhance the documentation with examples that can cover the :

- how the binding directory structure look like
- configuration for the build time
- configuration for the runtime
- using optional dependency-mapping 

Also its difficult to understand the ENV variable support which mentions NewRelic Extensions but does not give the context about what they are and how they are relevant in this context.

**Comment by pivotal-david-osullivan** [comment]
Thanks for opening the issue @kiranpatel11 - we can definitely make some improvements here. 

In case you haven't seen it, the APM section in the docs shows an example of creating a Binding and setting it at build & runtime: https://paketo.io/docs/howto/app-monitor/#example-with-a-binding . The example uses ApplicationInsights but NewRelic should be mostly the same. The full repo for the sample is here: https://github.com/paketo-buildpacks/samples/tree/main/java/application-insights

**Comment by anoopvlcy** [comment]
Hi @pivotal-david-osullivan 
As per Paketo [documentation](https://paketo.io/docs/howto/app-monitor/#new-relic) :
_"Connection credentials will be read from the binding at runtime, with the binding secrets being transformed to system properties with the following the pattern NEW_RELIC_<KEY>=<VALUE>."_

Does it mean that we need to create Binding secrets as follows?

 echo "$NR_TMP_KEY" > "binding/new-relic/LICENSE_KEY"
 echo "REFIMPL_TEST_NR" > "binding/new-relic/APP_NAME"
 echo "TEST_HOST" > "binding/new-relic/HOST_DISPLAY_NAME"
 echo "true" > "binding/new-relic/DISTRIBUTED_TRACING_ENABLED"

then during run time this will be converted to  NEW_RELIC_LICENSE_KEY=<VALUE>, NEW_RELIC_DISTRIBUTED_TRACING_ENABLED=<VALUE>  etc?
Im referring [NewRelic page](https://docs.newrelic.com/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#ev-NEW_RELIC_APP_NAME) for above mentioned environment variables.

Could you please clarify?

**Comment by pivotal-david-osullivan** [comment]
Yes that should be correct, could you please share your build & run commands as well as the output you are seeing?

**Comment by zaljic** [comment]
@pivotal-david-osullivan I am trying to use the new-relic buildpack as stated in the paketo binding docs, but the new-relic buildpack doesn't participate automatically. When specifying it with the `-b`flag, it participates. My license key is not being set in the newrelic.yaml though. Could you add a working example to the documentation? For some reason I am not getting it to work
